packages/overlays/src/OverlayController.js

```diff
@@ -498,8 +498,10 @@ export class OverlayController {
       case 'full':
         this._contentNodeWrapper.style.width = referenceWidth;
         break;
-      default:
+      case 'min':
         this._contentNodeWrapper.style.minWidth = referenceWidth;
+        break;
+      default:
```

Comment:
```suggestion
      /* no default */
```
will work for eslint 👍

Reply:
Done!